src/software/simulated_tests/example_play_test.cpp

```diff
@@ -15,6 +15,7 @@ class ExamplePlayTest : public SimulatedTest
 
 TEST_F(ExamplePlayTest, test_example_play)
 {
+    enableVisualizer();
```

Comment:
This will need to be removed again to pass CI. The servers CI runs on don't even have an X-window system (I think) so this will fail.

Reply:
done